### DISCUSSION TITLE
The Chia Holiday 2021 NFT

### DISCUSSION THREAD

**Discussion by ilciukalex88** [Chia-Network/chia-blockchain#14170] [Support]
hello, can you help, I really wanted to get NFT from your Christmas promotion, but I was not careful, I sent instead of CH 2021 to the address from your site, XCH :).
is it possible to fix this somehow?

xch1gj3r07889afmwjq6auk3ptthujl7ewyru74s45edd96lpue99g9smacgqy

**Comment by AndrewSavers** [comment]
Any updates?

**Comment by ilciukalex88** [comment]
xch1zqtndy2vunkpze6sqk70wdt5akm7qxc8vqwuwaumw4736efaqqnshz359s


вт, 20 дек. 2022 г. в 13:24, AndrewSavers ***@***.***>:
